app/utils/chart-config/versioning.ts

```diff
@@ -1,15 +1,22 @@
 import { DEFAULT_OTHER_COLOR_FIELD_OPACITY } from "@/charts/map/constants";
 import { ChartConfig, ConfiguratorState } from "@/config-types";
 import { mapValueIrisToColor } from "@/configurator/components/ui-helpers";
+import { FIELD_VALUE_NONE } from "@/configurator/constants";
 import {
   LEGACY_PROD_DATA_SOURCE_URL,
   PROD_DATA_SOURCE_URL,
 } from "@/domain/datasource/constants";
+import { client } from "@/graphql/client";
+import { joinIris } from "@/graphql/resolvers/rdf";
```

Comment:
I think the joinIris, splitIris methods should not come directly from the resolvers, this would be a concept that would be source agnostic, no ?